federation/client/clientset_generated/federation_internalclientset/typed/core/unversioned/fake/fake_core_client.go

```diff
@@ -46,8 +47,8 @@ func (c *FakeCore) Services(namespace string) unversioned.ServiceInterface {
 	return &FakeServices{c, namespace}
 }
 
-// GetRESTClient returns a RESTClient that is used to communicate
+// RESTClient returns a RESTClient that is used to communicate
 // with API server by this client implementation.
-func (c *FakeCore) GetRESTClient() *restclient.RESTClient {
-	return nil
+func (c *FakeCore) RESTClient() restclient.Interface {
+	return &fake.RESTClient{}
```

Comment:
Of course, I could update every place where ``client.Core().RESTClient()`` is used in tests as well and test for ``nil``. However, there are a lot of places which does not test the ``nil`` value.